packages/backend-for-frontend/src/routers/clientRouter.ts

```diff
@@ -233,7 +233,8 @@ const clientRouter = (
         const users = await clientService.getClientUsers(
           req.params.clientId,
           ctx.authData.selfcareId,
-          ctx
+          ctx,
+          req.query.name
```

Comment:
We usually leave context as the last parameter

```suggestion
          req.query.name,
          ctx,
```